scripts/ingest.rb

```diff
@@ -1,8 +1,10 @@
 require_relative '../lib/rails_stub'
 require_relative 'lib/pb_core_ingester'
-require 'logger'
+require_relative '../lib/has_logger'
```

Comment:
We haven't added `lib` directory to `$LOAD_PATH`, so you still need to `require_relative`, similar to requiring `rails_stub` above.